packages/webapp/src/components/Task/AbandonTask/styles.module.scss

```diff
@@ -19,8 +19,10 @@
     border: 1px solid var(--grey400);
     background-color: var(--grey200);
     color: var(--grey500);
-    margin-left: 4px;
+    margin-left: 6px;
     font-size: .875rem;
+    font-weight: 600;
+    line-height: 24px;
```

Comment:
Thank you for adding these.
I don't think line-height is working as you intended. I think one way to make it work is to set `display: inline-block;` and adjust its padding.

Reply:
Done the changes

Reply:
It looks like no padding top and button are needed...
I think this is a good time to check in with Carolina and get an approval from her, if you have not.

Reply:
Done, and i have requested Carolina to review it.

Reply:
Carolina mentioned that she would review it in beta, so I will go ahead and merge. (I personally think that it would be more efficient to share a screenshot with Carolina and get an approval before merging a PR in case she has a request. Sorry for lack of clarification)
Thank you @gursimran-singh!